distribution/std-lib/Standard/src/Image/Data/Histogram.enso

```diff
@@ -13,6 +15,7 @@ type Histogram
         Json.from_pairs [["data", bins]]
 
 ## Create a histogram for a specified channel from the image.
+calculate : Image -> Integer -> Histogram
 calculate image channel =
```

Comment:
I still think this should be done as an extension on the `Image` type.

Reply:
Ok, refactored as an extension.